docs/intro-to-coalton.md

```diff
@@ -491,16 +491,14 @@ COALTON-USER> (type-of '/)
 ∀ :A :B. DIVIDABLE :A :B ⇒ (:A → :A → :B)
 ```
 
-Because of the generic nature of division, if you're computing some values at the REPL, "raw division" simply does not work.
+Division defaults to `Double-Float` operations when given `Integer` values. [see Instance Defaulting](#instance-defaulting).
```

Comment:
```
Because of [instance defaulting](...), division of integer constants without any additional context defaults to `Double-Float` division.
```

or something like that.